### PR TITLE
add sso route to msb params

### PR DESCRIPTION
**Pull request by pb82**
Update to latest msbroker which adds a service instance for rhsso

### CODE REVIEW

**Comment by pb82** [comment]
@maleck13 it's deployed to my cluster: https://master.pbraun-73b1.openshiftworkshop.com/console/project/webapp/overview

When you open the webapp you should see the new RH-SSO route on the overview page. @david-martin has already verified that part.

File: roles/msbroker/tasks/main.yml [hunk 1]
```diff
@@ -73,6 +77,7 @@
   --param=THREESCALE_DASHBOARD_URL={{ threescale_dashboard_url }} \
   --param=APICURIO_DASHBOARD_URL={{ apicurito_dashboard_url }} \
   --param=SHARED_FUSE_DASHBOARD_URL={{ fuse_dashboard_url }} \
+  --param=SSO_URL=https://{{ sso_route.stdout }}/auth/admin/{{ sso_realm }}/console \
```
Comment (david-martin) [inline roles/msbroker/tasks/main.yml]: @pb82 This should be the `openshift` realm. Is that the case here?

Comment (pb82) [reply roles/msbroker/tasks/main.yml]: Yes, this is populated from https://github.com/integr8ly/installation/blob/master/roles/rhsso/defaults/main.yml#L6 through https://github.com/integr8ly/installation/pull/527/files#diff-4ea592a838e3c8063aa17dbe4ed16379R172

**Comment by maleck13** [comment]
Verified the url populated in MSB was https://sso-sso.apps.pbraun-73b1.openshiftworkshop.com/auth/admin/openshift/console

**Pull request merged**